### DISCUSSION TITLE
Understanding Browser Handler Arguments

### DISCUSSION THREAD

**Discussion by Cyb3r-Monk** [cmu-sei/GHOSTS#119] [Q&A] [answered]
I'm trying to understand what below parameters mean. I think they are applicable to the `crawl` command, but I can't figure out how they work. Is there an explanation? 

```JSON 
"HandlerArgs": {
        "stickiness": "65",
        "stickiness-depth-min": "3",
        "stickiness-depth-max": "15",
        "visited-remember": "5"
}
```

Also, regarding the trackable timeline, should I just give a define random guid for the `"TrackableId": "<guid id from trackables table/>"` or is there a trackables table that I should use?

**Comment by sei-dupdyke** [comment, marked as answer]
More applicable to a random browse command.

"stickiness": "65" 
        "stickiness-depth-min": "3",
        "stickiness-depth-max": "15",
        "visited-remember": "5"


Means that for a new site, 65% of the time, stay on that site and browse a random link found within this site, and do this between 3 and 15 times. Don't browse the same link of the last 5 URLs that you just browsed, which is useful for avoiding getting stuck in a main navigation loop.

**Reply by sei-dupdyke** [reply]
Trackables are for noting particular actions that you may want to act upon later. So if you have a trackable for browsing to a site, downloading some payload, and then executing that payload, a trackable keeps tabs on that flow. There is no table for these yet, you'd have to build some other system for managing them at the moment.

**Reply by Cyb3r-Monk** [reply]
Thanks for the info Dustin! 
Can I just put a random number manually for the trackables? I'm not using the API server, just one machine configured locally.

**Reply by sei-dupdyke** [reply]
As long as it is a valid UUID/GUID, yes.
